subprojects/packagefiles/sigdb/meson_options.txt

```diff
@@ -0,0 +1 @@
+option('install_dir', type: 'string', value: '', description: 'Install path for sigdb files')
```

Comment:
By setting the `value` here to the right directory, installing it with default settings into the prefix would work.